src/main/java/com/jakubroks/quiz/controller/ReportController.java

```diff
@@ -0,0 +1,25 @@
+package com.jakubroks.quiz.controller;
+
+import com.jakubroks.quiz.service.ReportService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/report")
+public class ReportController {
+
+
+    private final ReportService reportService;
+
+    public ReportController(ReportService reportService) {
+        this.reportService = reportService;
+    }
+
+    @GetMapping
+    public ResponseEntity<?> downloadReport() {
+        return null;
```

Comment:
może rozwiń ten endpoint zakończenie gry powinno zwrócić wynik bez raportu (trochę to co było w pierwszej wersji) pobranie raportu niech będzie osobnym endpintem (trzeba jakoś wskazać np. po uuid gry albo coś takiego) - ja bym zrobił takie flow:

gdy kończy się gra - użytkownik dostaje wynik końcowy (to co było), jego gra jest zapisywana do bazy, tak aby w przyszłości można było wygenerować raport) - jeżeli użytkownik chce generować raport to uderza na endpoint, taki raport się generuję i jest zwracany użytkownikowi (byte które zostały wygenerowane można odłożyć - np w bazie lub cacheu, bo jak użytkownik ponownie poprosi o ten sam raport, to nie trzeba go na nowo generować)